src/main/java/com/tasks/DoableTask.java

```diff
@@ -0,0 +1,52 @@
+package com.tasks;
+
+import com.util.json.JsonObject;
+import com.core.savedata.Schema;
+
+public class DoableTask extends Task {
+
+    /**
+     * Flag denoting task is done.
+     */
+    private boolean isDone;
+
+    /**
+     * Create instance of Task with isDone flag.
+     *
+     * @param taskName name of task
+     */
+    public DoableTask(String taskName) {
+        super(taskName);
+        isDone = false;
+    }
+
+    /**
+     * Set isDone to true.
+     */
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    /**
+     * Get icon based on isDone status.
+     *
+     * @return icon string
+     */
+    private String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
+
+    /**
+     * get string representation of task.
+     *
+     * @return string
+     */
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + super.toString();
+    }
```

Comment:
Nice idea of overriding the toString() method to help with the printing of tasks 👍